v0.1/options/axes.schema.json

```diff
@@ -15,25 +22,22 @@
       "description": "Metadata node identifier.",
       "pattern": "sands/options/axes/([A-Za-z0-9_-]+).json"
     },
-    "origin": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "origin"
-    },
-    "units": {
+    "anatomicalOrientation": {
       "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "units"
+      "description": "Axes orientation denoted in standard anatomical terms of location."
     },
-    "directions": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "directions"
+    "origin": {
+      "type": "array",
+      "description": "Center of the corodinate system (point where axes intersect).",
+      "items": {
+        "type": "number"
+      },
+      "minItems": 3,
+      "maxItems": 3
     },
-    "anatomicalLabels": {
+    "unit": {
       "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "anatomicalLabels"
-    },
-    "indexingParameters": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/INDEXING_PARAMETERS",
-      "description": "indexingParameters"
+      "description": "Standard measure that is used to describe the axes."
```

Comment:
I would be more general for the general properties or being more specific. 
E.g., more generic: "A determinate quantity adopted as a standard of the measurement."
E.g., more specific: "The determinate length quantity that defines a single step along each axes."
[maybe you find better ones; I would go for more generic (this is true also for all other often used properties, such as alias...)]

Reply:
You do not have to change this for this PR

Reply:
Okay, I noted it done. When I update the type/ID stuff, I'll change it to the generic suggestions (which I think is appropriate).